horreum-client/src/main/java/io/hyperfoil/tools/HorreumClient.java

```diff
@@ -183,22 +181,23 @@ public HorreumClient build() throws IllegalStateException {
             ResteasyClient client = clientBuilder.build();
             ResteasyWebTarget target = client.target(horreumUrl);
 
-            return new HorreumClient(client,
-                  target.proxyBuilder(ActionService.class).build(),
-                  target.proxyBuilder(AlertingService.class).build(),
-                  target.proxyBuilder(BannerService.class).build(),
-                  target.proxyBuilder(ChangesService.class).build(),
-                  target.proxyBuilder(ConfigService.class).build(),
-                  target.proxyBuilder(DatasetService.class).build(),
-                  target.proxyBuilder(ExperimentService.class).build(),
-                  target.proxyBuilder(NotificationService.class).build(),
-                  target.proxyBuilder(ReportService.class).build(),
-                  new RunServiceExtension(target, target.proxyBuilder(RunService.class).build()),
-                  target.proxyBuilder(SchemaService.class).build(),
-                  target.proxyBuilder(SqlService.class).build(),
-                  target.proxyBuilder(SubscriptionService.class).build(),
-                  target.proxyBuilder(TestService.class).build(),
-                  target.proxyBuilder(UserService.class).build());
+            return new HorreumClient(
+                    client, target.proxyBuilder(ActionService.class).build(), target.proxyBuilder(
+                            AlertingService.class
+                    ).build(), target.proxyBuilder(BannerService.class).build(), target.proxyBuilder(
+                            ChangesService.class
+                    ).build(), target.proxyBuilder(ConfigService.class).build(), target.proxyBuilder(
+                            DatasetService.class
+                    ).build(), target.proxyBuilder(ExperimentService.class).build(), target.proxyBuilder(
+                            NotificationService.class
+                    ).build(), target.proxyBuilder(ReportService.class).build(), new RunServiceExtension(
+                            target, target.proxyBuilder(RunService.class).build()
+                    ), target.proxyBuilder(SchemaService.class).build(), target.proxyBuilder(
+                            SqlService.class
+                    ).build(), target.proxyBuilder(SubscriptionService.class).build(), target.proxyBuilder(
+                            TestService.class
+                    ).build(), target.proxyBuilder(UserService.class).build()
```

Comment:
I find this format strange, formatting alternate parameters differently

Reply:
+1, it is an issue but I think I found the proper option.. will change it in a while

Reply:
With the latest change, this is formatted as:
```java
            return new HorreumClient(
                    client,
                    target.proxyBuilder(ActionService.class)
                            .build(),
                    target.proxyBuilder(AlertingService.class)
                            .build(),
                    target.proxyBuilder(BannerService.class)
                            .build(),
                    target.proxyBuilder(ChangesService.class)
                            .build(),
                    target.proxyBuilder(ConfigService.class)
                            .build(),
                    target.proxyBuilder(DatasetService.class)
                            .build(),
                    target.proxyBuilder(ExperimentService.class)
                            .build(),
                    target.proxyBuilder(NotificationService.class)
                            .build(),
                    target.proxyBuilder(ReportService.class)
                            .build(),
                    new RunServiceExtension(
                            target,
                            target.proxyBuilder(RunService.class)
                                    .build()
                    ),
                    target.proxyBuilder(SchemaService.class)
                            .build(),
                    target.proxyBuilder(SqlService.class)
                            .build(),
                    target.proxyBuilder(SubscriptionService.class)
                            .build(),
                    target.proxyBuilder(TestService.class)
                            .build(),
                    target.proxyBuilder(UserService.class)
                            .build()
            );
```
which seems much better